coalib/output/ConsoleInteraction.py

```diff
@@ -641,8 +641,8 @@ def show_bear(console_printer, bear, sections, metadata):
                      "Supported languages:",
                      bear.LANGUAGES,
                      "  ",
-                     "The bear does not provide information about which "
-                     "languages it can analyze.")
+                     "(The bear does not provide information about which "
+                     "languages it can analyze.)")
```

Comment:
essentially this string is shown if there's no info about this property, we have that below for the used sections, needed parametres etc. they should have all the same style which is why I made a function that shows it. If we want them in brackets that function should add the brackets consistently automatically.